docs/porting-guides/6.x.x.md

```diff
@@ -150,6 +158,59 @@ snmp_settings:
       ipv4_acl: IPV4
 ```
 
+#### Migrating from CloudVision keys to `cv_settings`
+
+In the previous versions of AVD, you may have defined your CloudVision settings as follows.
+
+=== "Previous Variables"
+
+    ```yaml
+    cvp_instance_ips:
+      - 192.168.200.12
+      - 192.168.200.13
+    cvp_token_file: /tmp/token
+    terminattr_disable_aaa: true
+    terminattr_ingestexclude: /Sysdb/cell/1/agent,/Sysdb/cell/2/agent
+    terminattr_ingestgrpcurl_port: 9910
+    terminattr_smashexcludes: ale,flexCounter,hardware,kni,pulse,strata
+    ```
+
+=== "Rendered Configuration"
+
+    ```eos
+    daemon TerminAttr
+      exec /usr/bin/TerminAttr -cvaddr=192.168.200.12:9910,192.168.200.13:9910 -cvauth=token,/tmp/token -cvvrf=default -disableaaa -smashexcludes=ale,flexCounter,hardware,kni,pulse,strata -ingestexclude=/Sysdb/cell/1/agent,/Sysdb/cell/2/agent -taillogs
+      no shutdown
+    ```
+
+These keys are now removed and replaced with `cv_settings`, to build a similar output, you should update your variables with the following structure.
+
+=== "AVD 6.x"
+
+    ```yaml
+    cv_settings:
+      onprem_clusters:
+        - name: MAIN
+          servers:
+            - name: 192.168.200.12
+            - name: 192.168.200.13
+      terminattr:
+        disable_aaa: true
+        ingestexclude: /Sysdb/cell/1/agent,/Sysdb/cell/2/agent
+        smashexcludes: ale,flexCounter,hardware,kni,pulse,strata
+    ```
+
+=== "Rendered Configuration"
+
+    ```eos
+    daemon TerminAttr
+      exec /usr/bin/TerminAttr -cvaddr=192.168.200.12:9910,192.168.200.13:9910 -cvauth=token,/tmp/token -cvvrf=default -disableaaa -smashexcludes=ale,flexCounter,hardware,kni,pulse,strata -taillogs -cvsourceintf=Management0 -smashexcludes=ale,flexCounter,hardware,kni,pulse,strata -ingestexclude=/Sysdb/cell/1/agent,/Sysdb/cell/2/agent
```

Comment:
why do we have dupicate smash exclude here? is it what we are rendering?

Reply:
Fixed it.

Reply:
thanks!